### PR TITLE
Fix missing download progress on taskbar (Issue #516)

### PR DESCRIPTION
**Pull request by Gpower2**
Used new DownloadsTaskbar.jsm from Mozilla

### CODE REVIEW

File: browser/components/downloads/moz.build [hunk 1]
```diff
@@ -14,5 +14,6 @@ EXTRA_COMPONENTS += [
 
 EXTRA_JS_MODULES += [
     'DownloadsCommon.jsm',
-    'DownloadsLogger.jsm',
+	'DownloadsLogger.jsm',
```
Comment (wolfbeast) [inline browser/components/downloads/moz.build]: Nit: please be mindful of your whitespace. Use spaces, not tabs, and keep things aligned, please.

File: toolkit/mozapps/downloads/moz.build [hunk 2]
```diff
@@ -17,11 +17,8 @@ EXTRA_PP_COMPONENTS += [
 EXTRA_JS_MODULES += [
     'DownloadLastDir.jsm',
     'DownloadPaths.jsm',
+	'DownloadTaskbarProgress.jsm',
```
Comment (wolfbeast) [inline toolkit/mozapps/downloads/moz.build]: Ditto

File: browser/base/content/browser.js [hunk 3]
```diff
@@ -1068,13 +1068,10 @@ var gBrowserInit = {
...
     setTimeout(function() {
       try {
-        let DownloadsCommon =
-          Cu.import("resource:///modules/DownloadsCommon.jsm", {}).DownloadsCommon;
-        DownloadsCommon.initializeAllDataLinks();
-        // TODO: fix taskbar-based download progress
-        //let DownloadTaskbarProgress =
-        //  Cu.import("resource://gre/modules/DownloadTaskbarProgress.jsm", {}).DownloadTaskbarProgress;
-        //DownloadTaskbarProgress.onBrowserWindowLoad(window);
+		Cu.import("resource:///modules/DownloadsCommon.jsm", {})
```
Comment (wolfbeast) [inline browser/base/content/browser.js]: Ditto

**Comment by Gpower2** [comment]
Replaced tabs with whitespaces

**Comment by wolfbeast** [comment]
:+1: Great! Thank you :)
Another issue to cross of the to-do for v27. We're making progress!

**Pull request merged**